src/operations.py

```diff
@@ -43,7 +43,7 @@ def __repr__(self):
 class OpHandler:
 
     @staticmethod
-    def getMsgForOp(source: SingleItemNode, op: Op, type: messages.MsgType=messages.MsgType.Request, dest=""):
+    def getMsgForOp(source: SingleItemNode, op: Op, dest=""):
```

Comment:
did you intend to change this? i.e. does the type function parameter need to go?

Reply:
I got a bug when type was included and couldn't run it.